src/libraries/System.Net.Http/tests/StressTests/HttpStress/NuGet.config

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!-- Add public nuget feed. -->
-    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <clear />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
```

Comment:
Will all the packages we use be there? Could you run the stress pipeline on this PR? It's not part of inner loop.